### DISCUSSION TITLE
Can't open xml file after saving and closing the file

### DISCUSSION THREAD

**Discussion by ozzybayman** [harborsiem/WindowsRibbon#35] [Q&A]
I already make xml file and make some tab in there and when it tested it works fine but after saving and closing app when I open the file again it says

ERROR: Index and length must refer to a location within the string.
Parameter name: length

and I can't open the file unless I delete all of tab on view section

**Comment by harborsiem** [comment]
Please send XML File .

**Comment by ozzybayman** [comment]
[RIBBONUI.zip](https://github.com/user-attachments/files/16563722/RIBBONUI.zip)
